backend/dal/Migrations/Initial/Up/PostUp/01-BuildingPredominantUses.sql

```diff
@@ -74,6 +74,6 @@ INSERT INTO dbo.[BuildingPredominateUses] (
     , 0
 ), (
     17
-    , 'ZEBRA FEEDING STATION'
+    , 'Marina'
```

Comment:
I'm assuming this will work if my other change goes through first?

Reply:
This doesn't do anything for production, or any DB that is already running.  But it does add it if a database is refreshed or setup for the first time.